src/qtcreatorplugin.pri

```diff
@@ -95,7 +96,8 @@ include(rpath.pri)
 contains(QT_CONFIG, reduce_exports):CONFIG += hide_symbols
 
 TEMPLATE = lib
-CONFIG += plugin plugin_with_soname
+!os2:CONFIG += plugin plugin_with_soname
+os2:CONFIG += shared dll
```

Comment:
Thi one shouldn't be necessary. We should check how Windows handles it and do the same.

Reply:
windows code uses plugin. The issue could be related to gcc: I'm sure gcc was able to import symbols directly from a .dll while linking, maybe this does no longer work. plugin config does not generate an import lib.